src/EnergyPlus/OutputProcessor.cc

```diff
@@ -687,35 +577,42 @@ namespace OutputProcessor {
                                                                      cNumericFieldNames);
 
             // Check for duplicates?
+            auto &reqRepVar = op->ReqRepVars(Loop);
+            reqRepVar.Key = cAlphaArgs(1);
+            if (reqRepVar.Key == "*") {
+                reqRepVar.Key = std::string();
+            }
 
-            op->ReqRepVars(Loop).Key = cAlphaArgs(1);
-            if (op->ReqRepVars(Loop).Key == "*") {
-                op->ReqRepVars(Loop).Key = std::string();
+            bool is_simple_string = !UtilityRoutines::isKeyRegexLike(reqRepVar.Key);
+            reqRepVar.is_simple_string = is_simple_string;
+            if (!is_simple_string) {
+                reqRepVar.case_insensitive_pattern = std::make_shared<RE2>("(?i)" + reqRepVar.Key);
             }
 
+            // TODO: trim any resulting trailing whitespace?
             std::string::size_type const lbpos = index(cAlphaArgs(2), '['); // Remove Units designation if user put it in
             if (lbpos != std::string::npos) {
                 cAlphaArgs(2).erase(lbpos);
             }
```

Comment:
This is existing code, so I didn't do it (yet at least). But if I put something like:

`Output:Variable,*,Zone Mean Air Temperature [C],hourly;` then I end up with `Zone Mean Air Temperature ` (trailing space) as a key and it'll never match, won't it?

Reply:
Right trim implemented in 0dccd1d1838d3115b89898a709d7345efeb27fe2